suites/upgrade/infernalis/older/3-upgrade-sequence/upgrade-mon-osd-mds.yaml

```diff
@@ -0,0 +1,44 @@
+meta:
+   - desc: |
+      upgrade mon a and b
```

Comment:
'upgrade the ceph cluster nodes' - mon.a and mon.b are the two cluster nodes, upgrading them upgrades everything there.

description of restart order is fine